content/gsoc/2025/_index.md

```diff
@@ -60,3 +60,45 @@ The type system design will require particular attention to several key areas. T
 * [Robert Timms](https://github.com/rtimms)
 * [Arjun Verma](https://arjxnpy.vercel.app/)
 * [Ferran Brosa Planella](https://www.brosaplanella.xyz/) -->
+
+## Mutation testing PyBaMM with mutmut
+
+Mutation testing checks your tests' effectiveness by making minor modifications to the codebase and running the test suite. The tests are not specific or good enough if they pass with the changes made by the mutation testing framework. [mutmut](https://mutmut.readthedocs.io) is a mutation testing library for Python that has recently been adopted in testing suite of large projects.
+
+PyBaMM (Python Battery Mathematical Modelling) has a considerably huge testing suite, which often ends up having weak or flaky tests that bump up the coverage but do not test the code thoroughly. Mutation testing would be an effective way to check whether the tests work. Moreover, this is an exploratory project, and the mentee will actively make design decisions that will pave the way for either adoption or rejection of occasional mutation testing of PyBaMM. Finally, the mentee will be guided by the mentors to fix the problems identified by mutmut, thus making the test suite better with each run.
+
+### Technical details
+
+mutmut is an intelligent testing framework with a browser/terminal TUI making it easy to identify "mutants" and apply them using simple commands. The library also remembers the tests that have been checked so that you can fix them incrementally. The mentee will need to use the browser/terminal TUI interactively, find bad tests, report them, and possibly fix them.
+
+mutmut can also execute tests smartly and parallelly, making the execution fast. Once the mentee is familiar with the entire workflow, they can utilize the parallel execution capabilities to speed up the workflow and report failures quickly.
```

Comment:
```suggestion
mutmut can also execute tests smartly and in parallel, making the execution fast. Once the mentee is familiar with the entire workflow, they can utilize the parallel execution capabilities to speed up the workflow and report failures quickly.
```

i.e., "parallelly" ➡️ "in parallel"; the former is less idiomatic ;) https://english.stackexchange.com/a/15373